pipelines/_ingredients/ApPipe.yaml

```
@@ -72,6 +72,7 @@ tasks:
     config:
       doConfigureApdb: False
       apdb_config_url: parameters.apdb_config
+      alertPackager.useAveragePsf: True  # Not added to ApPipeWithFakes
```

Commit message could be a bit clearer; maybe:
```suggestion
      alertPackager.useAveragePsf: True  # Speed up production processing; don't want as default or in ApPipeWithFakes
```